<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginWidget</class>
 <widget class="QWidget" name="loginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="mainWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1340</width>
     <height>780</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1340</width>
      <height>780</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{border-image: url(:/init);}</string>
    </property>
    <widget class="QLabel" name="week">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>420</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="date">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>500</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="startbtn">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>1311</width>
       <height>761</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="minibtn">
     <property name="geometry">
      <rect>
       <x>1240</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="closebtn">
     <property name="geometry">
      <rect>
       <x>1290</x>
       <y>10</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="accInput">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>380</y>
       <width>334</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>请输入账户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwdInput">
     <property name="geometry">
      <rect>
       <x>238</x>
       <y>464</y>
       <width>334</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);
border-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>请输入密码</string>
     </property>
    </widget>
    <widget class="QLabel" name="week1">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>410</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="date1">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>500</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="loginbtn">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>590</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="errorMsg">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>530</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>16</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>*账户或密码错误</string>
     </property>
    </widget>
    <zorder>week</zorder>
    <zorder>date</zorder>
    <zorder>minibtn</zorder>
    <zorder>closebtn</zorder>
    <zorder>accInput</zorder>
    <zorder>passwdInput</zorder>
    <zorder>week1</zorder>
    <zorder>date1</zorder>
    <zorder>loginbtn</zorder>
    <zorder>errorMsg</zorder>
    <zorder>startbtn</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
